golem_messages/message/tasks.py

```diff
@@ -251,6 +254,8 @@ class TaskToCompute(ConcentEnabled, TaskMessage):
         'size',  # the size of the resources zip file
         'concent_enabled',
         'price',  # total subtask price computed as `price * subtask_timeout`
+
+        '_ethsig'  # must be last
```

Comment:
Why is that? Is it because of some backward compatibility where it may or may not be set, thus having different length? It's going to always be set eventually, right? Why implement handling of the non set case?

Reply:
@Krigpl no, rather so that it's not treated as part of the message itself (in which case, the signature would be part of the payload it then signs)